CoreFoundation/String.subproj/CFString.h

```diff
@@ -147,9 +147,6 @@ equivalents in the encoding the compiler will use to interpret them (for instanc
 O-umlaut is \303\226 in UTF-8). UTF-8 is the recommended encoding here, 
 since it is the default choice with Mac OS X developer tools.
 */
-#if TARGET_OS_WIN32
-#undef __CONSTANT_CFSTRINGS__
```

Comment:
im sure this might require compiler changes

Reply:
I took care of those changes a while ago.  constant CFStrings are generated properly on Windows by clang.  Unfortunately, gcc and cl are not viable options due to the ABI level changes for supporting the CFString to swift.String bridging.